src/ducks/metadata.js

```diff
@@ -62,20 +57,9 @@ export const convertField = (nameAttr, convertType) => ({
   convertType,
 });
 
-export const updateTitle = title => ({
-  type: UPDATE_TITLE,
-  title,
-});
-
-export const updateBody = body => ({
-  type: UPDATE_BODY,
-  body,
-});
-
-export const updatePath = path => ({
-  type: UPDATE_PATH,
-  path,
-});
+export const updateTitle = title => ({ type: UPDATE_TITLE, title });
```

Comment:
One-line is not always good. I had the same feeling years ago but today I find them unnecessary. They are not only hard to read but also hard to make modifications. For example, if we need to do a calculation inside this function, we have to add paranthesis and return statement at the end. Another think is we don't have any concern on too many LOCs or big filesizes. All of the code is minified at the end.

Reply:
okay..

Reply:
> All of the code is minified at the end.

True. But the sourcemap isn't minified. That's what increases the size of the published gem.

Reply:
Hmm. We shouldn't need sourcemap for production though.

Reply:
That is up for debate. However, the following might help:
https://css-tricks.com/should-i-use-source-maps-in-production/